### PR TITLE
New Valdiations

### PR DESCRIPTION
**Pull request by Jedi2002**
Added new Validationds based on the document 
https://docs.google.com/document/d/1nDRob1pQm5Csp3vlxyibNpUWn47wdiCkwFUsR7sRcHM/edit

Still not a a 100 % on the discussion around the identifiers 
https://docs.google.com/document/d/1nDRob1pQm5Csp3vlxyibNpUWn47wdiCkwFUsR7sRcHM/edit?disco=AAAAKzP-yC4
https://docs.google.com/document/d/1nDRob1pQm5Csp3vlxyibNpUWn47wdiCkwFUsR7sRcHM/edit?disco=AAAAHTCcKVU
https://docs.google.com/document/d/1nDRob1pQm5Csp3vlxyibNpUWn47wdiCkwFUsR7sRcHM/edit?disco=AAAAHSSd990

This can be added in later as part of a future task

### CODE REVIEW

**Review by khdegraaf — APPROVED** [review]
Looks fine from a test perspective.

File: x/clp/genesis_test.go [hunk 1]
```diff
@@ -71,6 +71,8 @@ func CreateState(ctx sdk.Context, keeper clp.Keeper, t *testing.T) (int, int) {
 	for _, lp := range lpList {
 		keeper.SetLiquidityProvider(ctx, lp)
 	}
+	V1 := test.GenerateValidatorAddress("A58856F0FD53BF058B4909A21AEC019107BA6")
```
Comment (banshee) [inline x/clp/genesis_test.go:74]: Can we extract A58856F0FD53BF058B4909A21AEC019107BA6 to a constant?  We use it in a lot of places.

**Review by AustinoBombino — COMMENTED** [review]
looks good to me

**Review by banshee — APPROVED** [review]

**Review by utx0 — APPROVED** [review]
ok LGTM. I have however left some more comment in the validation google doc and we should talk about that and workout the next steps.

**Pull request merged**